src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $box-gap;
+  width: 7 * $box-size + 6 * $box-gap;
+
+  @for $lastDay from 28 through 31 {
+    &--month-length--#{$lastDay}
+    &__day:nth-child(n + #{$lastDay + 1}) {
+      display: none;
+    }
+  }
+
+  @each $dayName, $number in $days {
+    &--start-day--#{$dayName}
+    &__day:first-child {
+      margin-left: ($number * $box-size + $number * $box-gap);
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    height: $box-size;
+    width: $box-size;
+    border: $box-gap solid black;
+    background-color: #eee;
+    transition: background-color 0.5s, transform 0.5s;
+
+    @for $number from 1 through 31 {
+      &:nth-child(#{$number})::before {
+        content: "#{$number}";
+        font-family: Arial, sans-serif;
```

Comment:
You can set font-family to the parent tag and it will work. Read about inheritance